corehq/apps/commtrack/static/commtrack/ko/locations.js

```diff
@@ -44,7 +25,7 @@ function LocationSettingsViewModel() {
 
     this.new_loctype = function() {
         var parent_pk = (_.last(settings.loc_types()) || {}).pk;
-        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, this);
+        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, commtrack_enabled);
```

Comment:
I initially set `commtrack_enabled` on the `LocationSettingsViewModel` (the context in which this method is called), so it could be accessed from `root.commtrack_enabled` in the `LocationTypeModel`.  Turns out `root` wasn't used anywhere in that model, and it wasn't actually a reference to the `LocationSettingsViewModel` anyways (it needed `var self = this;`, then pass `self` in instead).  I figured instead I'd just pass in a boolean and reduce the state tying the two models together, since `commtrack_enabled` is the only thing shared right now anyways.

The `settings` variable is, I think, somewhat poorly named - all it contains is the list of location types.  I think the original author thought of "settings" as equivalent to "location types", as that's also the name of this template.  I see now that `settings` is actually a dict containing one key `loc_types`.  I could add another key to that dict called `commtrack_enabled`, is that what you're suggesting?  `commtrack_enabled` is already used in the template, so I just made it available in the JS as well.